views/sms/index.js

```diff
@@ -241,7 +241,7 @@ exports.respond = function(req, res) {
             return save_utils.countRequestsPerRegion(region_code);
         }).then( function(numRequests) {
             requestData = save_utils.createSerial(numRequests, req.cookies.request_object, region_code);
-            requestData.is_sms = 'sms';
+            requestData.origin = 'sms';
```

Comment:
Whups, sorry, I missed that subtlety.  But yes, I think just making everything `source` would be best in the long run.